### PR TITLE
toffy documentation standardization

### PR DESCRIPTION
**Pull request by ngreenwald**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**
Gives a more comprehensive description of the purpose of each notebook in the repo. Reorganizes and names most used notebooks to emphasize use cases. 

**Remaining issues**
Still need to update the specific folder names to be created

### CODE REVIEW

File: templates/1_set_up_toffy.ipynb [hunk 1]
```diff
@@ -5,28 +5,51 @@
    "id": "ec35cb18-c771-48e6-859f-e10e074545d6",
```
Comment (ackagel) [inline templates/1_set_up_toffy.ipynb:5]: <p>we could also pull an ark-analysis here and have an update script to automatically do this + manage a git-tracked templates vs untracked scripts folder.  maybe a PR for another time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by ackagel — COMMENTED** [review]
looks good. just a few comments on some possible future additions

**Review by alex-l-kong — COMMENTED** [review]
Nothing else to add on top of @ackagel.

**Comment by srivarra** [comment]
https://github.com/angelolab/toffy/blob/b31f7fbf4e682d2d62c504d673a4ce918fab1d93/templates/2_create_tiled_mibi_run.ipynb?short_path=252b0ba#L100

The Moly point insertion link doesn't seem to work.

**Comment by camisowers** [comment]
Instructions seem pretty clear to me!

**Pull request merged**